### ISSUE TITLE
[prefer-input/output/model/view-child/content-child-readonly] rule proposal

### ISSUE THREAD

**Issue by MillerSvt** [angular-eslint/angular-eslint#1892]
**Description of the proposal**

Enforce readonly for Input Signal, Model Input, Output, Content Queries

**Incorrect code**
```typescript
class MyComp {
   foo = input();
   bar = model();
   baz = output();
   element = viewChild('element');
   elements = viewChildren('element');
   content = contentChild('content');
   contents = contentChildren('content');
}
```

**Correct code**
```typescript
class MyComp {
   readonly foo = input();
   readonly bar = model();
   readonly baz = output();
   readonly element = viewChild('element');
   readonly elements = viewChildren('element');
   readonly content = contentChild('content');
   readonly contents = contentChildren('content');
}
```

**Comment by MillerSvt** [comment]
I can do this. Wait for approve.
And I need to know what name of new rule you want?

**Comment by MillerSvt** [comment]
Maybe it should be done for all signals (e.g. signal(), computed()), what do you think?

**Comment by P4** [comment]
FYI, there's an open PR https://github.com/angular-eslint/angular-eslint/pull/1872 implementing `prefer-signal-readonly`, you might be interested in looking at that one

**Comment by MillerSvt** [comment]
This seems to be a duplicate of #1719.
